src/focus-ring.js

```diff
@@ -74,11 +80,13 @@ document.addEventListener('DOMContentLoaded', function() {
    * opening a menu or dialog.
    */
   function onKeyDown() {
-    hadKeyboardEvent = true;
     // `activeElement` defaults to document.body if nothing focused,
     // so check the active element is actually focused.
-    if (matches(document.activeElement, ':focus'))
-      addFocusRingClass(document.activeElement);
+    var activeElement = document.activeElement;
+    if (activeElement.tagName == 'BODY')
+      return;
+
+    hadKeyboardEvent = true;
```

Comment:
Because the more I thought about it, we don't want to set that to true if focus hasn't moved to a focusable control. This way we can use `hadKeyboardEvent` to determine if a focus moved to a focusable element as a result of keyboard navigation.